roles/sufia_dependencies/install/defaults/main.yml

```diff
@@ -11,3 +11,5 @@ hydra_pg_pass: my_hydra_pg_pass
 # needed for resque-pool init script
 capistrano_user: deploy
 fits_version: '0.8.4'
+# on debian, package version is now 9.4
+postgres_path: /etc/postgresql/9.3/main
```

Comment:
This was a quick-and-dirty workaround and could use a refactor. What is a better way to address the pgsql version? We could add the postgres repo and specify a version to apt-get. I left 9.3 as the default because I still think of ubuntu as the default OS.